stardoc/stardoc.bzl

```diff
@@ -53,6 +53,14 @@ def _stardoc_impl(ctx):
         omit_if_empty = True,
         uniquify = True,
     )
+    # Needed in case some files are referenced across local repository
+    # namespace. For example, consider a file under a local repository @bar
+    # which may be referenced as either //foo/bar:lib.bzl or @bar//:lib.bzl.
```

Comment:
"which may be referenced as either //foo/bar:lib.bzl" - really?

```
~/code/rules_jvm_external
$ bazel query //examples/android_local_test:*
ERROR: no such package 'examples/android_local_test': Package is considered deleted due to --deleted_packages
```

This is the case for me regardless whether there's a local_repository pointing to android_local_test in the parent workspace or not.

where `examples/android_local_test/WORKSPACE` exists.

Reply:
Aha. You're right, you can't actually build these targets otherwise across the boundary. I was mistaken for the reasoning.

I've updated the comment to better reflect what I'm seeing.